modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -112,9 +107,9 @@ spec:
   isRdma: true
 ----
 
-The following example describes the configuration for a SR-IOV device in OpenStack Virtual Machine:
+The following example describes the configuration for an SR-IOV device in a {rh-openstack} virtual machine:
```

Comment:
I noticed the platform is using {rh-openstack-first} in <13>, but this one is {rh-openstack}, is there any difference?